### DISCUSSION TITLE
replaced LLM by MoE model

### DISCUSSION THREAD

**Discussion by Nurburgring-Zhang** [lllyasviel/Omost#73] [Ideas]
To expand, replaced LLM by MoE model, and each individual small model corresponds to a different art style. When drawing each time, different units and weights are called to combine different styles, which can not only reduce the performance requirements of LLM, but also increase more varied styles.

**Comment by QijiTec** [comment]
同上，扩展一下，用MoE模型取代了LLM，每个单独的小模型对应着不同的艺术风格。每次绘图时，调用不同的单位和权重来组合不同的风格，这样不仅可以降低LLM的性能要求，还可以增加更多样的风格。
